articles/virtual-machines/shared-image-galleries.md

```diff
@@ -101,7 +101,7 @@ The properties of an image version are:
 
 There are two operating system states supported by Azure Compute Gallery. Typically images require that the VM used to create the image has been [generalized](generalize.yml) before taking the image. Generalizing is a process that removes machine and user specific information from the VM.  For Linux, you can use [waagent](https://github.com/Azure/WALinuxAgent) `-deprovision` or `-deprovision+user` parameters. For Windows, the Sysprep tool is used.
 
-Specialized VMs haven't been through a process to remove machine specific information and accounts. Also, VMs created from specialized images don't have an `osProfile` associated with them. This means that specialized images will have some limitations in addition to some benefits.
+Specialized VMs haven't been through a process to remove machine specific information and accounts. Specialized VM image contains operating system disk and all the attached disks. Also, VMs created from specialized images don't have an `osProfile` associated with them. This means that specialized images will have some limitations in addition to some benefits.
```

Comment:
@Court72 - Same thing here as well, generalized and specialized both contain os and data disks (if any). so I don't think this statement is needed.